src/main/java/seedu/address/flashcard/Choice.java

```diff
@@ -1,23 +1,41 @@
 package seedu.address.flashcard;
 
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
 /**
  * Represents a MCQ choice.
  */
 public class Choice {
 
-    private final String content;
```

Comment:
I initially named it as `content` because I thought it would reduce confusion from having too many "choice" names.
But `choice` works fine too.

Reply:
I see! I think I can change it to `value` in that case, I changed it to `choice` to standardize with the other classes but it seems `content` or `value` (which was used by the initial AB) would be clearer.

Reply:
Updated in the next commit, can you help me to take a look @joshtyf?